compiler/parser.nim

```diff
@@ -1448,10 +1486,10 @@ proc postExprBlocks(p: var Parser, x: PNode): PNode =
 proc parseExprStmt(p: var Parser): PNode =
   #| exprStmt = simpleExpr
   #|          (( '=' optInd expr colonBody? )
-  #|          / ( expr ^+ comma
+  #|          / ( exprEqExpr ^+ comma
   #|              postExprBlocks
   #|            ))?
-  var a = simpleExpr(p)
+  var a = simpleExpr(p, pmCommandStart)
```

Comment:
`pmCommandStart` here is not covered by the grammar, which is a bit important, but personally I don't think the added complexity to the grammar is worth it. The `exprEqExpr` part was an existing part of the syntax that wasn't documented.